recipe/meta.yaml

```diff
@@ -12,9 +12,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py2k]
+  skip: true  # [py2k or py<=36]
 
 requirements:
   build:
```

Comment:
`setuptools >=40` and `wheel` should be in `host`, see https://github.com/aio-libs/yarl/blob/68e7d9312ad9eccb695dbc174a4705e9a238c1fa/pyproject.toml#L2

Reply:
There is a typo in the fix (should be `40`, not `4.0`).

Reply:
additionally specifying in skip py2k and skipping for anything below py<37 is overspecified.  Having just py<37 should be enough

Reply:
@katietz  that's been corrected in the latest fixup

@marcoesters fixed, thanks